docs/getting-started/live-networks.md

```diff
@@ -16,25 +16,37 @@ Regen Mainnet launched with the `v1.0.0` release tag of `regen-ledger`.
 
 <!-- TODO: update to use dedicated full node operated by RND -->
 
-The following URL is the node endpoint for one of our lead validators:
+The following URL is the RPC endpoint for one of our lead validators:
 
 [http://104.131.169.70:26657/](http://104.131.169.70:26657/)
 
 For more information, see [regen-network/mainnet](https://github.com/regen-network/mainnet).
 
-### Regen Devnet
+### Redwood Testnet
 
-`regen-devnet-5` is the chain ID for Regen Devnet.
+`regen-redwood-1` is the chain ID for Redwood Testnet.
 
 <!-- TODO: add information about genesis binary and upgrade binaries -->
 
-Regen Devnet launched with the `v1.0.0-rc0` release tag of `regen-ledger`.
+Redwood Testnet launched with the `v1.0.0` release tag of `regen-ledger`.
 
-<!-- TODO: update to use dedicated full node operated by RND -->
+The following URL is the RPC endpoint for a dedicated full node operated by RND:
+
+[http://redwood.regen.network:26657/](http://redwood.regen.network:26657/)
+
+For more information, see [regen-network/testnets](https://github.com/regen-network/testnets).
+
+### Hambach Testnet
+
+`regen-hambach-1` is the chain ID for Hambach Testnet.
 
-The following URL is the node endpoint for one of our lead validators:
+<!-- TODO: add information about genesis binary and upgrade binaries -->
```

Comment:
is this TODO captured in some existing issue?

Reply:
opened #566